remote/src/main/scala/org/apache/pekko/remote/transport/netty/NettyHelpers.scala

```diff
@@ -36,7 +35,7 @@ private[netty] trait NettyHelpers {
 
   protected def onOpen(@unused ctx: ChannelHandlerContext): Unit = ()
 
-  protected def onMessage(@unused ctx: ChannelHandlerContext, @unused msg: ByteBuf): Unit = ()
+  protected def onMessage(@unused ctx: ChannelHandlerContext, @unused msg: Array[Byte]): Unit = ()
```

Comment:
Is there anything special about ByteBuf ? (;I don't know about Netty

Reply:
This PR is just a Try, and will not be needed. the old code is actually right.